docker/Dockerfile-python3

```diff
@@ -4,6 +4,15 @@ FROM python:3.6.10
 
 RUN apt-get update && apt-get install -y wget build-essential curl nginx supervisor
 
+# get nodejs install script and pass it to execute: 
+RUN curl -sL https://deb.nodesource.com/setup_12.x | bash
+# and install nodejs
+RUN apt-get install nodejs
+
+# install newman and newman-reporter-html
+RUN npm install -g newman
+RUN npm install -g newman-reporter-html
```

Comment:
All of this can be chained in one run command using '&&' so we can have less build steps